conda/cli/main_remove.py

```diff
@@ -55,12 +54,11 @@ def execute(args, parser):
     if args.all:
         if prefix == context.root_prefix:
             raise CondaEnvironmentError(
-                "cannot remove root environment,\n"
-                "       add -n NAME or -p PREFIX option"
+                "cannot remove root environment, add -n NAME or -p PREFIX option"
             )
         if not isfile(join(prefix, "conda-meta", "history")):
             raise DirectoryNotACondaEnvironmentError(prefix)
-        print("\nRemove all packages in environment %s:\n" % prefix, file=sys.stderr)
+        print(f"\nRemove all packages in environment {prefix}:\n")
```

Comment:
changes stderr to stdout

Reply:
correct, IMO the text printed does not belong in stderr:

```bash
$ conda create -n scratch python --yes
...
$ conda remove -n scratch --all --yes > /dev/null

Remove all packages in environment /Users/kodegard/.conda/arm64/23.1.0/3.10/envs/scratch:

```

```bash
$ conda create -n scratch python --yes
...
$ conda remove -n scratch --all 2> /dev/null

## Package Plan ##

  environment location: /Users/kodegard/.conda/arm64/23.1.0/3.10/envs/scratch


The following packages will be REMOVED:

  bzip2-1.0.8-h620ffc9_4
  ca-certificates-2023.01.10-hca03da5_0
  certifi-2022.12.7-py311hca03da5_0
  libffi-3.4.2-hca03da5_6
  ncurses-6.4-h313beb8_0
  openssl-1.1.1t-h1a28f6b_0
  pip-23.0.1-py311hca03da5_0
  python-3.11.2-hc0d8a6c_0
  readline-8.2-h1a28f6b_0
  setuptools-65.6.3-py311hca03da5_0
  sqlite-3.41.2-h80987f9_0
  tk-8.6.12-hb8d0fd4_0
  tzdata-2023c-h04d1e81_0
  wheel-0.38.4-py311hca03da5_0
  xz-5.2.10-h80987f9_1
  zlib-1.2.13-h5a0b063_0


Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```

Reply:
yeah that seemed to be a bug